### DISCUSSION TITLE
xmake v2.6.6

### DISCUSSION THREAD

**Discussion by waruqi** [xmake-io/xmake#2379] [Announcements]
### New features

* [#2327](https://github.com/xmake-io/xmake/issues/2327): Support nvc/nvc++/nvfortran in nvidia-hpc-sdk
* Add path instance interfaces
* [#2344](https://github.com/xmake-io/xmake/pull/2344): Add lz4 compress module
* [#2349](https://github.com/xmake-io/xmake/pull/2349): Add keil/c51 project support
* [#274](https://github.com/xmake-io/xmake/issues/274): Distributed compilation support
* Use builtin local cache instead of ccache

### Changes

* [#2309](https://github.com/xmake-io/xmake/issues/2309): Support user authorization for remote compilation
* Improve remote compilation to support lz4 compression

### Bugs fixed

* Fix lua stack when select package versions

### 新特性

* [#2327](https://github.com/xmake-io/xmake/issues/2327): 支持 nvidia-hpc-sdk 工具链中的 nvc/nvc++/nvfortran 编译器
* 添加 path 实例接口
* [#2344](https://github.com/xmake-io/xmake/pull/2344): 添加 lz4 压缩模块
* [#2349](https://github.com/xmake-io/xmake/pull/2349): 添加 keil/c51 工程支持
* [#274](https://github.com/xmake-io/xmake/issues/274): 跨平台分布式编译支持
* 使用内置的本地缓存替代 ccache

### 改进

* [#2309](https://github.com/xmake-io/xmake/issues/2309): 远程编译支持用户授权验证
* 改进远程编译，增加对 lz4 压缩支持

### Bugs 修复

* 修复选择包版本时候 lua 栈不平衡导致的崩溃问题

<hr /><em>This discussion was created from the release <a href='https://github.com/xmake-io/xmake/releases/tag/v2.6.6'>xmake v2.6.6</a>.</em>

**Comment by shuax** [comment]
#2334 和lz4有关系吗？是不是写错了。

**Reply by waruqi** [reply]
github 的 bug， 里面的 link 实际是 pr 的，不是 issues，github 跳转错了。。`https://github.com/xmake-io/xmake/pull/2334`

**Comment by shuax** [comment]
那是2344，不是2334

**Reply by waruqi** [reply]
哦 改了
